prompt/prompt-executor/prompt-executor-clients/prompt-executor-bedrock-client/src/jvmMain/kotlin/ai/koog/prompt/executor/clients/bedrock/BedrockLLMClient.kt

```
@@ -527,20 +556,42 @@ public class BedrockLLMClient(
                                         is AttachmentContent.Binary.Base64 -> source = Bytes(imageContent.asBytes())
                                         is AttachmentContent.Binary.Bytes -> source = Bytes(imageContent.data)
                                         is AttachmentContent.PlainText ->
-                                            source =
-                                                Bytes(imageContent.text.encodeToByteArray())
-
-                                        else -> {}
+                                            source = Bytes(imageContent.text.encodeToByteArray())
+                                        else -> {
+                                            logger.warn { "Unknown image content type: ${imageContent::class.simpleName}" }
```

Let's be more strict and throw here instead of logging?